routes/accounts.js

```diff
@@ -270,6 +270,44 @@ function accountRoutes (server, options, next) {
     }
   }
 
+  var postAccountsSessionRoute = {
```

Comment:
I've started this in the accounts routes file since it's related. I'm wondering if I should create a separate file for this.